coderd/workspaceagents.go

```diff
@@ -1463,6 +1473,96 @@ func (api *API) workspaceAgentsExternalAuthListen(ctx context.Context, rw http.R
 	}
 }
 
+// @Summary User-scoped agent coordination
+// @ID user-scoped-agent-coordination
+// @Security CoderSessionToken
+// @Tags Agents
```

Comment:
drop this tag --- it's not specifically related to agents, unlike the existing RPC which targets a specific agent.

Reply:
The swagger parser requires a tag, so it'd be between `Agents` or `Workspaces`, or perhaps a new tag?